lib/saluki-components/src/sources/checks/scheduler.rs

```diff
@@ -238,7 +257,8 @@ impl Scheduler {
             for _ in 0..(desired_count - current_count) {
                 self.add_worker();
             }
-            info!("Increased worker count to {}", desired_count);
+
+            info!(worker_count = desired_count, "Check worker count updated to.");
```

Comment:
Ahh, looks like I left an extra word in my original suggestion:

```suggestion
           info!(worker_count = desired_count, "Check worker count updated.");
```